doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_application.rst

```diff
@@ -0,0 +1,874 @@
+:orphan:
+
+.. _migration_cs3_to_2_6_99_cs2_app:
```

Comment:
```suggestion
.. _migration_cs3_to_2_6_99_cs2_app:
```

@nvlsianpu @kl-cruz @grochu I'd need some detailed inputs about this file. Every line currently listed here is just a placeholder to give the idea, but might be technically incorrect. Could you please check this and provide some feedback regarding the differences with 2.4.99-cs3?

Reply:
I think there haven't been any changes in public API in the scope of my team between these releases, but we'll look at the details and report if anything is missing.